finesse/hardware/plugins/time/host_time.py

```diff
@@ -0,0 +1,15 @@
+"""This module provides an interface for querying localhost system time."""
+from time import time
+
+from finesse.hardware.plugins.time.time_base import TimeBase
+
+
+class HostTime(TimeBase, description="Host time"):
+    """A time source that queries the system time."""
+
+    def close(self) -> None:
+        """Close the connection to the device."""
+
+    def get_time(self) -> float:
```

Comment:
What the actual output (a float) means? Is it time from epoch or what? Maybe add some comment on that.

Reply:
Yeah, seconds since the epoch. I'll update the comments.